analyzer/src/funTest/assets/projects/synthetic/spdx/package/libs/curl/package.spdx.yml

```diff
@@ -18,10 +18,11 @@ packages:
      IMAP, SMTP, POP3, RTSP and RTMP. libcurl offers a myriad of powerful features."
   copyrightText: "Copyright (c) 1996 - 2020, Daniel Stenberg, <daniel@haxx.se>, and many
     contributors, see the THANKS file."
-  downloadLocation: "git+github.com:curl/curl.git@53cdc2c963e33bc0cc1a51ad2df79396202e07f8"
+  downloadLocation: "git+github.com/curl/curl.git@53cdc2c963e33bc0cc1a51ad2df79396202e07f8"
   filesAnalyzed: false
   homepage: "https://curl.haxx.se/"
   licenseConcluded: "NOASSERTION"
   licenseDeclared: "curl"
   name: "curl"
+  packageFileName: "."
```

Comment:
packageFileName is optional in the SPDX spec see [1] why are you adding it here?

[1] https://github.com/spdx/spdx-spec/blob/development/v2.2.1/chapters/package-information.md#74-package-file-name-field-

Reply:
Per the original spec for SPDX manifest https://github.com/spdx/spdx-spec/issues/439#issue-643990957 packageFileName is only used for project.spdx.yml to indicate the subdir the package is in but it seems when we implemented in ORT we also added it to package.spdx.yml with value "." to indicate the current directory. Not sure of this is intent with SPDX spec but it is valid syntax